ynr/apps/ynr_refactoring/admin.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+
+from .models import PageNotFoundLog
+
+
+@admin.register(PageNotFoundLog)
+class PageNotFoundLogAdmin(admin.ModelAdmin):
```

Comment:
I think want we really want to see here is more like totals grouped by URL, not just a raw list i.e: `SELECT url, COUNT(*) FROM log GROUP BY url;` not `SELECT * FROM log;`. Particularly given every 404 is basically the same: we're not recording anything unique to each case like the user who triggered it or anything.

When I was working on the reports for the EE admin, I read quite lot of tutorials on customising the django admin. I reckon this one tells you what you need to know: https://medium.com/@hakibenita/how-to-turn-django-admin-into-a-lightweight-dashboard-a0e0bbf609ad (and a bunch of other stuff)

Probably not worth spending loads of time on this as its only short-term, but would be quite a nice improvement for future-Chris and future-Sym if its quick :)